modules/ztp-intro-factory-install.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: CONCEPT
+[id="introduction-factory-install_pipeline_{context}"]
+= An introduction to the factory install pipeline
+
+The factory install pipeline builds out your factory environment (hub and edge clusters) in readiness for the edge cluster to be shipped off site. Red Hat has created a set of community scripts to help you get started with this task.
+
+A GitHub repository contains all the relevant scripts and YAML files you need to provision the hub cluster and edge clusters.
+
+The edge cluster installation uses a zero touch provisioning (ZTP) approach facilitated by {rh-rhacm-first} using the Assisted Installer (AI) installed as part of running the factory install pipeline.
+
+With ZTP and AI, you can provision many {product-title} edge clusters in a factory-type setting. RHACM manages clusters in a hub and edge architecture, where a single hub cluster manages many edge or spoke clusters. A hub cluster running ACM provisions and deploys the edge clusters using ZTP and AI. The AI service handles provisioning of {product-title} of the edge clusters running on bare metal.
```

Comment:
You have a note above "In the documentation and particularly with reference to the various scripts invoked you may see the term spoke cluster or spoke clusters used. The preferred term to use in relation to ZTPFWs is edge cluster or edge clusters and they effectively mean the same thing."

So in "many edge or spoke clusters" the latter part is redundant.

Reply:
I agree mention one place as looks clunky repeating